modules/virtualization/microvm/netvm.nix

```diff
@@ -38,7 +38,7 @@
         microvm.interfaces = [
           {
             type = "tap";
-            id = "vm-netvm";
+            id = "vm-${config.networking.hostName}";
```

Comment:
My 2 cents for aligning the `netvm` filename and documentation name with the system declared `id` - so `netvm` instead of this line.

Reply:
I thought a bit about this and decided this should be also out of scope for this PR since it touches other files, and the logic of that work does not correspond with the aim of this PR.
Moreover, changing `netvm` id somehow breaks the netvm.
I'll track this issue as the part of SP-2839 ticket.